crates/subspace-networking/Cargo.toml

```diff
@@ -67,11 +67,10 @@ features = [
     "serde",
     "tcp",
     "tokio",
-    "websocket",
     "yamux",
 ]
 [dependencies.libp2p-quic]
-version = "0.8.0-alpha"
+version = "0.9.2"
```

Comment:
There is a `quic` feature on `libp2p` crate again so you don't need to add it explicitly anymore.